client/my-sites/stats/stats-post-detail/index.jsx

```diff
@@ -86,6 +113,18 @@ class StatsPostDetail extends Component {
 				/>
 
 				<PostWeeks siteId={ this.props.siteId } postId={ this.props.postId } />
+				<WebPreview
```

Comment:
The only issue I see with the `WithPreviewProps` approach is we lose the ability to have the Edit button in the WebPreview toolbar.  I might just rely upon a combination of `get( site, 'is_previewable' ) && ! isJetpack` and conditionally use WebPreview or an external link.